examples/lusid/instruments/upload-term-deposit-instruments.sql

```diff
@@ -0,0 +1,38 @@
+-- ============================================================
+
+-- Description:
+-- 1. In this query, we run an ETL process on some instruments.
+-- 2. First, we load a CSV file of instruments from Drive.
+-- 3. Next, we transform the shape of the instrument data.
+-- 4. Finally we upload the instrument data into LUSID.
+
+-- ============================================================
+
+-- Extract Term Deposit data from LUSID Drive
+
+@term_deposit_file = use Drive.Excel
+--file=/luminesce-examples/instruments.xlsx
+--worksheet=term_deposits
+enduse;
+
+-- Transform data using SQL
+
+@term_deposit_instruments = select

```

Comment:
The data from the excel file has column headings that perfectly match the required inputs for the writer, this is easier but would it be more confusing for someone using the example as they may think they don't need to do mapping.
I know it would serve material purpose but would have the below be clearer?

@term_deposit_instruments = select
ClientInternal as ClientInternal,
Name as Name... etc.

Reply:
Sure, updated.